pluggy.py

```diff
@@ -581,7 +595,7 @@ def varnames(func):
         try:
             func = func.__init__
         except AttributeError:
-            return ()
+            return (), ()
```

Comment:
i just noticed something dreadful, since this is a public name, this is a breaking change would warrant a major incompatible release